__tests__/Auth0Client/getTokenSilently.test.ts

```diff
@@ -1605,11 +1605,10 @@ describe('Auth0Client', () => {
       const auth0 = setup();
 
       await loginWithRedirect(auth0);
-
       mockFetch.mockReset();
-
       jest.spyOn(auth0, 'logout');
-      jest.spyOn(utils, 'runIframe').mockRejectedValue(
+
+      const utilSpy = jest.spyOn(utils, 'runIframe').mockRejectedValue(
```

Comment:
Nothing critical, but I was confused about the name here (utilSpy sounds a bit generic so I was about to propose `runIframeSpy`, but I cant see it being used so we might as well revert this)

Reply:
Good spot, I think it'll be reverted 👍🏻